go.mod

```diff
@@ -29,9 +31,11 @@ require (
 	github.com/go-swagger/go-swagger v0.25.0
 	github.com/go-test/deep v1.0.7
 	github.com/gofrs/uuid v3.3.0+incompatible // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
not directly related to the review, but that's an interesting library :)

Reply:
The great thing about standard libraries like protobuf is that there are so many to choose from :-(